nebula/ui/components/VPNTutorialList.qml

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+import QtQuick.Controls 2.15
+
+import Mozilla.VPN 1.0
+import Mozilla.VPN.qmlcomponents 1.0
+import components 0.1
+
+GridLayout {
+    property var customTutorialFilter: () => true
+    property var count: tutorialRepeater.count
+
+    Layout.alignment: Qt.AlignTop
```

Comment:
In a reusable components like this (and typically root level components in general), you usually want to avoid using `Layout.` or `anchors.` because you don't know whether it will actually be inside a layout (where you CAN use `Layout.`) or a positioner (where you CAN use `anchors.`).

You should no longer need this if you also implement [this comment](https://github.com/mozilla-mobile/mozilla-vpn-client/pull/4221/files/4b464c41374fb57dd2b80483790cffaf01a29454#r955650594)

Reply:
Makes total sense, thanks for catching this.